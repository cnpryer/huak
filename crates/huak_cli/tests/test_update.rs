#[cfg(test)]
mod tests {
    use insta_cmd::assert_cmd_snapshot;
    use std::process::Command;

    #[test]
    fn test_help() {
        assert_cmd_snapshot!(Command::new("huak").arg("update").arg("--help"));
    }
}
