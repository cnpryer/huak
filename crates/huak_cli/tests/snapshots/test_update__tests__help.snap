---
source: crates/huak_cli/tests/test_update.rs
info:
  program: huak
  args:
    - update
    - "--help"
---
success: true
exit_code: 0
----- stdout -----
Update the project's dependencies

Usage: huak update [OPTIONS] [DEPENDENCIES]... [-- <TRAILING>...]

Arguments:
  [DEPENDENCIES]...  
  [TRAILING]...      Pass trailing arguments with `--`

Options:
  -q, --quiet  
  -h, --help   Print help

----- stderr -----

